packages/composites/the-graph/README.md

```diff
@@ -0,0 +1,117 @@
+# Chainlink The-graph Composite Adapter
+
+The Graph adapter is a generic adapter to query information from The Graph.  It currently only supports fetching prices from the Uniswap subgraph.
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+| Required? |   Name    |         Description          | Options | Defaults to |
+| :-------: | :-------: | :--------------------------: | :-----: | :---------: |
+|    ✅     | `RPC_URL` | The RPC Url to connect to |         |             | 
+|           | `UNISWAP_V2_SUBGRAPH_ENDPOINT` | The Uniswap V3 subgraph endpoint |         |     https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2        |
```

Comment:
> The Uniswap V3 subgraph endpoint
Is this v2 or v3 subgraph, or are they interchangeable?

Reply:
V2, the V3 one has different request/response objects.